### PR TITLE
Document manual Redis failover procedure

### PR DESCRIPTION
**Pull request by anothertobi**

### CODE REVIEW

**Review by mhutter — APPROVED** [review]

File: docs/modules/ROOT/pages/how-tos/redis/manual_failover.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+sentinel Could not connect to Redis at 10.0.0.20:26379: Connection refused
+----
+
+To resolve this situation, get the IP address of the running leader, connect to a Redis Sentinel Pod, start a `redis-cli` and initiate the failover.
+
+[source,shell]
+----
+kubectl get pod -o wide
+
+kubectl exec redis-node-0 sentinel -it -- sh
```
Comment (mhutter) [inline docs/modules/ROOT/pages/how-tos/redis/manual_failover.adoc:18]:
```suggestion
kubectl exec redis-node-0 -c sentinel -it -- sh
```

**Review by mhutter — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/how-tos/redis/manual_failover.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+redis-cli -a $(< $REDIS_PASSWORD_FILE) -p 26379
```
Comment (mhutter) [inline docs/modules/ROOT/pages/how-tos/redis/manual_failover.adoc:19]: Does also not work:

```
mh $ kubectl exec redis-node-0 -c sentinel -it -- sh
$ redis-cli -a $(< $REDIS_PASSWORD_FILE) -p 26379
Warning: Using a password with '-a' or '-u' option on the command line interface may not be safe.
Warning: AUTH failed
(error) ERR unknown command `26379`, with args beginning with:
$
$
```

```suggestion
export REDISCLI_AUTH="$(< $REDIS_PASSWORD_FILE)"
redis-cli -p 26379
```

**Review by mhutter — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/how-tos/redis/manual_failover.adoc (hunk 2 above)
Comment (mhutter) [reply docs/modules/ROOT/pages/how-tos/redis/manual_failover.adoc:19]: Oh, `< file` only works in `bash`, so either replace `<` with `cat` or `sh` (above) with `bash`

File: docs/modules/ROOT/pages/how-tos/redis/manual_failover.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+In certain situations Redis nodes end up in `CrashLoopBackOff` because they receive a wrong (old) leader IP address from the running Sentinel.
+
+In the logs this might look like:
+
+[source,shell]
+----
+sentinel Could not connect to Redis at 10.0.0.20:26379: Connection refused
+----
+
+To resolve this situation, get the IP address of the running leader, connect to a Redis Sentinel Pod, start a `redis-cli` and initiate the failover.
```
Comment (mhutter) [inline docs/modules/ROOT/pages/how-tos/redis/manual_failover.adoc:12]: >get the IP address of the running leader

How?

Sorry, only realized this just now...

Comment (anothertobi) [reply docs/modules/ROOT/pages/how-tos/redis/manual_failover.adoc:12]: With the `kubectl get pod -o wide` the IP is in the output. Should I change the sentence somehow to make this clear?

**Review by mhutter — APPROVED** [review]

**Pull request merged**